app/src/main/res/layout/fragment_movies_list.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/background"
+    tools:context=".activities.movie_list.MoviewListFragment">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingBottom="@dimen/edge_margin_large">
+
+        <TextView
+            android:id="@+id/title"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/edge_margin"
+            android:layout_marginTop="@dimen/edge_margin_large"
+            android:drawablePadding="8dp"
+            android:includeFontPadding="false"
+            android:paddingVertical="12dp"
+            android:text="@string/movies_list"
+            android:textColor="@color/white"
+            android:textSize="14dp"
+            android:textStyle="bold"
+            app:drawableStartCompat="@drawable/ic_movie_list"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:ignore="SpUsage" />
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
**REQUIRED**
nested constrant layout not good for perfomance

Reply:
this nested constraint is temporary, i think there will be some of listView or recyclerView in next homeworks, so it'll be removed

Reply:
@LeoHoneyBeard okay 👍